### PR TITLE
SAK-49615 Lessons: Title of a Question Item incorrectly displayed

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-49615

### CODE REVIEW

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ReorderProducer.java [hunk 1]
```diff
@@ -249,7 +249,7 @@ public void fillComponents(UIContainer tofill, ViewParameters params, ComponentC
 				    // oembed. use the URL
 				    UILink.make(row, "link", i.getAttribute("multimediaUrl"), i.getAttribute("multimediaUrl"));
 				} else if (i.getType() == SimplePageItem.QUESTION) {
-				    String text = i.getAttribute("questionText");
+				    String text = i.getAttribute("questionCorrectText");
```
Comment (ottenhoff) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ReorderProducer.java:252]: what is questionCorrectText?

Comment (kunaljaykam) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ReorderProducer.java:252]: `questionCorrectText` is the original question text without the `<p>` tag, The `<p>` gets added from the Iframe I think.  The span tag won't work with `<p>`:  https://forum.bootstrapstudio.io/t/unable-to-add-p-tags-inside-a-span-tag/6149

Comment (ottenhoff) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ReorderProducer.java:252]: are you sure?!

public String questionText, questionCorrectText, questionIncorrectText;

i assume questionCorrect and questionIncorrect refer to correct and incorrect feedback

Comment (kunaljaykam) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ReorderProducer.java:252]: Please let me check.

**Comment by kunaljaykam** [comment]
@ottenhoff Do you think, we should strip off the HTML tags from CKEditor directly before saving it?

**Comment by ottenhoff** [comment]
no, HTML is valid and ok

**Review by ottenhoff — APPROVED** [review]
Thanks

**Pull request merged**